appium/common/helper.py

```diff
@@ -17,7 +17,7 @@
 from appium import version as appium_version
 
 
-def appium_bytes(value, encoding):
+def appium_bytes(value: str, encoding: str) -> str:
```

Comment:
the method returns `bytes`

Reply:
Thanks, I've removed `appium_bytes` since it's for the compatibility for py2 and py3.